digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/ViiteApi.scala

```diff
@@ -1011,7 +1011,7 @@ class ViiteApi(val roadLinkService: RoadLinkService, val vVHClient: KgvRoadLink,
         pathParam[Long]("projectID").description("Id of a project")
       )
       tags "ViiteAPI - RoadAddresses"
-      summary "Returns all the road names that are related to a certain project (referenced by the projectID) and within a certain roadNumber."
+      summary "Returns a road name that is related to within a certain roadNumber or a certain project (referenced by the projectID)."
```

Comment:
"Returns a road name that is related to a certain roadNumber or a certain project (referenced by the projectID)."